setup.py

```diff
@@ -17,7 +17,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # http://packaging.python.org/en/latest/tutorial.html#version
-    version='2.1.43',
+    version='2.1.46',
```

Comment:
Changes in azure-pipelines.yml look good. There is no need to modify the version in setup.py, pull the latest changes from the develop branch if your branch is behind.